_data/download-descriptions.yml

```diff
@@ -198,6 +202,8 @@ linux-ubuntu-xenial-i386.deb:
   description: Linux Ubuntu Xenial 16.04 (i386, 32 bit)
 macosx-i686.zip:
   description: Mac OS X 10.4-10.5 (Intel build)
+macosx-x86_64.dmg:
```

Comment:
Should we rename this to "MacOS 10.12+ (Intel) (Apple Disk Image)", or only make that change if/when we deploy ARM support in one way or another?

Should there also be a macosx-x86_64.zip or is that still covered by macosx.zip?

Reply:
I did add the `Intel` part initially, but removed it :) Let's add that when we add ARM. I also think at that point we should remove `Apple Disk Image`, like .. we only publish 1 file for Intel, so that is not helpful :D

We produce no `macosx-x86_64.zip`. I don't know if you plan on adding that with the ARM stuff?

Reply:
Hmm, https://www.openttd.org/downloads/openttd-releases/latest.html shows both a .dmg and a .zip. I don't think we need both to be honest, but the .zip is a bit smaller. The .dmg is probably the Mac standard though.

Reply:
I can look at adding it back in if we want to keep it, but equally happy to go .dmg only. It looks like we can switch to a version of dmg that uses the bzip2 algorithm, which should then compress better than zip. Might offset some of the extra size due to universal binaries. ;)

Reply:
We did for 1.10, with the old `configure` stuff. CPack doesn't produce a `zip` (by default? Haven't looked into it). So for now we will only produce a `dmg`, until someone hugs CPack to add `zip`, if that is what we want :)

Reply:
CPack can produce a `zip` (I briefly enabled it and broke the compile farm).
https://github.com/OpenTTD/OpenTTD/pull/8189 also did some things for `zip`